Merge Conflict/Assets/Scripts/AudioManager/AudioManager.cs

```diff
@@ -0,0 +1,218 @@
+/**********************************************************************************************************************
+Name:          AudioManager
+Description:   Manages all sounds and background music that will be played.  
+Author(s):     Daniel Rittrich
+Date:          2024-03-24
+Version:       V1.3
+TODO:          - /
+**********************************************************************************************************************/
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioManager : MonoBehaviour
+{
+    private static AudioManager _instance;
+    public static AudioManager Instance { get { return _instance; } }
+
+    [Header("Playlist BG Music")]
+    public Playlist playlist;
+    public PlayMode playMode;
+
+    [Header("Music")]
+    public AudioSource backgroundAudioSource;
+    public AudioSource gameAudioSource;
+    public AudioSource menuAudioSource;
+
+    [Header("Effects")]
+    public AudioSource buttonClickAudioSource; // ?  Maybe separate AudioSources for different buttons
+    public AudioClip buttonClickSound;
+
+    [Space(20)]
+    public AudioSource openMenuAudioSource;
+    public AudioClip openMenuSound;
+
+    [Space(20)]
+    public AudioSource closeMenuAudioSource;
+    public AudioClip closeMenuSound;
+
+    [Space(20)]
+    public AudioSource mergeAudioSource;  // ?  Maybe separate sounds for lvl  2, 3, 4 
+    public AudioClip[] mergeSound;
+
+    [Space(20)]
+    public AudioSource combineComponentsAudioSource;
+    public AudioClip combineComponentsSound;
+
+    [Space(20)]
+    public AudioSource sellAudioSource;
+    public AudioClip sellSound;
+    public AudioClip trySellWrongComponentSound;
+
+    [Space(20)]
+    public AudioSource throwAwayAudioSource; // trash
+    public AudioClip throwAwaySound;
+
+    [Space(20)]
+    public AudioSource dropComponentAudioSource;
+    public AudioClip dropComponentSound;
+
+    [Space(20)]
+    public AudioSource pickUpComponentAudioSource;
+    public AudioClip pickUpComponentSound;
+
+    [Space(20)]
+    public AudioSource footstepAudioSource;
+    public AudioClip footstepSound;
+    public float footstepInterval = 0.18f;
+
+    // ?  Maybe other sounds or environment sounds 
+    /*    (conveyorbelt, component walking, component dragging, component dropping, individual sounds for upgrades, 
+           open menu, close menu, component returning to desk, trash walkin on desk, level up, close menu, quest completed,
+           get component when throwing one away (upgrade), buy component, selling wrong component,  ... )  */
+
+    // TODO: don't play dropComponent Sound when selling a component or throwing it in the trash can
+
+    private Coroutine playNextSongCoroutine;
```

Comment:
make it more clear that "nextSong" refers to the background music. This includes the rest of the "song" methods.